components/automate-cli/pkg/verifyserver/constants/checkconstant.go

```diff
@@ -14,33 +18,121 @@ const AWS_OPENSEARCH_S3_BUCKET_ACCESS = "aws-opensearch-s3-bucket-access"
 const EXTERNAL_POSTGRESQL = "external-postgresql"
 const NFS_BACKUP_CONFIG = "nfs-backup-config"
 
-const HARDWARE_RESOURCE_COUNT_MSG = "Hardware Resource Count Check"
-const SSH_USER_MSG = "SSH User Access Check"
-const CERTIFICATE_MSG = "Certificate check"
+const (
+	HARDWARE_RESOURCE_COUNT_MSG         = "Hardware Resource Count Check"
+	CERTIFICATE_MSG                     = "Certificate Check"
+	SSH_USER_MSG                        = "SSH User Access Check"
+	SYSTEM_RESOURCES_MSG                = "System Resources Check"
+	SOFTWARE_VERSIONS_MSG               = "Software Versions Check"
+	SYSTEM_USER_MSG                     = "System User Check"
+	S3_BACKUP_CONFIG_MSG                = "S3 Backup Config Check"
+	FQDN_MSG                            = "FQDN Check"
+	FIREWALL_MSG                        = "Firewall Check"
+	EXTERNAL_OPENSEARCH_MSG             = "External Opensearch Check"
+	AWS_OPENSEARCH_S3_BUCKET_ACCESS_MSG = "AWS Opensearch S3 Bucket Access Check"
+	EXTERNAL_POSTGRESQL_MSG             = "External Postgresql Check"
+	NFS_BACKUP_CONFIG_MSG               = "NFS Backup Config Check"
+)
 
 const NFS_MOUNT_MSG = "NFS Mount Backup Check"
 
+const (
+	BASTION_CHECK = "bastion"
+	REMOTE_CHECK  = "remote"
+)
+
+var checkMap map[string]models.CheckAndType = map[string]models.CheckAndType{
+	HARDWARE_RESOURCE_COUNT: {
+		CheckName: BASTION_CHECK,
```

Comment:
CheckType should hold the type of check i.e. Bastion or Remote
and CheckName should hold the name of the check. 
You swapped both the fields